cli/js/event_target_test.ts

```diff
@@ -60,8 +71,11 @@ test(function anEventTargetCanBeSubclassed(): void {
 
 test(function removingNullEventListenerShouldSucceed(): void {
   const document = new EventTarget();
+  // @ts-ignore
```

Comment:
again, what are we ignoring and why?

Reply:
Same issue as https://github.com/denoland/deno/pull/3899#discussion_r377594434